go/flags/endtoend/vttablet.txt

```diff
@@ -34,6 +34,12 @@ Usage of vttablet:
       --ceph_backup_storage_config string                                Path to JSON config file for ceph backup storage. (default "ceph_backup_config.json")
       --compression-engine-name string                                   compressor engine used for compression. (default "pargzip")
       --compression-level int                                            what level to pass to the compressor. (default 1)
+      --config-file string                                               Full path of the config file (with extension) to use. If set, --config-path, --config-type, and --config-name are ignored.
```

Comment:
Time to deprecate `--tablet_config`?
https://github.com/vitessio/vitess/pull/6070#issue-600485280

Reply:
sure; worth noting here that due to the order of operations (which isn't really avoidable, i think), the old `--tablet_config` gets applied _after_ the viper config loading, so any values in there take precedence.

i'd strongly advise (and will do so in the deprecation, which i think should be a separate change also in v17) that people use one or the other but never both.